plan/plan.go

```diff
@@ -229,6 +229,9 @@ type PhysicalPlan interface {
 	// getChildReqProps gets the required property by child index.
 	getChildReqProps(idx int) *requiredProp
 
+	// StatsCount returns the count of statsInfo for this plan.
+	StatsCount() float64
```

Comment:
I think this function can be removed.

Reply:
This is called by `executor` package to determine if the query is expensive.